primedev/windows/libsys.cpp

```diff
@@ -56,15 +56,27 @@ HMODULE WINAPI WLoadLibraryExA(LPCSTR lpLibFileName, HANDLE hFile, DWORD dwFlags
 	// replace xinput dll with one that has ASLR
 	if (lpLibFileName <= lpLibName && !strncmp(lpLibName, XINPUT1_3_DLL, strlen(XINPUT1_3_DLL) + 1))
 	{
-		hModule = o_LoadLibraryExA("XInput9_1_0.dll", hFile, dwFlags);
+		const char* replacementDll = "XInput1_4.dll";
```

Comment:
There's no need for `replacementDll`, just pass the string directly to the function.

Reply:
It's used for the log at the end

Reply:
Missed that, would be cool if you renamed it to `pszReplacementDll` <3

Reply:
i did it :3